### DISCUSSION TITLE
I cant sign in with my Github Profile.

### DISCUSSION THREAD

**Discussion by Mozgiii9** [GitbookIO/community#681] [0. General bugs and questions]
Always when i want to sign in on Gitbook with my Github profile, i have an error like on screen. What can i do? I tried to disconnect my Github profile, but it not working...![image](https://github.com/user-attachments/assets/cb9f3192-1dc6-413f-97ee-e45b1f04b197)

**Comment by aikoii** [comment]
Hey @Mozgiii9, sometimes, accounts are flagged automatically for various reasons like account sharing, VPN use, suspicious content, etc. Could you share your user ID or email at [support@gitbook.com](mailto:support@gitbook.com) 🙂
